kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/services/KmService.java

```diff
@@ -148,4 +152,25 @@ public void onFailure(String response, Exception e, Context context) {
     public static void updateChannel(Context context, GroupInfoUpdate groupInfoUpdate, AlChannelUpdateTask.AlChannelUpdateListener listener) {
         new AlChannelUpdateTask(context, groupInfoUpdate, listener).execute();
     }
+
+    /**
+     * the wrapper method to get feedback for given conversation
+     * @param conversationId the groupId of the conversation
+     * @return the response object, response.getData() will return null in case of feedback not found
+     */
+    public synchronized String getConversationFeedback(String conversationId) {
+        String response = clientService.getConversationFeedback(conversationId);
+        Utils.printLog(context, "KmService","Feedback get: "+response);
+        return response;
+    }
+
+    /**
+     * the wrapper method tp set the feedback for the conversation at the server
+     * @param kmFeedback the feedback object (has groupId, rating and comments data members)
+     * @return string response of the post request
+     */
+    public synchronized String setConversationFeedback(KmFeedback kmFeedback) throws Exception {
+        return clientService.setConversationFeedback(kmFeedback.getGroupId(), kmFeedback.getRating(), kmFeedback.getComments());
```

Comment:
@shubhamtewari by naming convention set methods are not supposed to return anything, please correct the naming convention.